.travis.yml

```diff
@@ -7,18 +7,16 @@ jobs:
     - stage: prechecks
       python: 3.6
       env: TOXENV=pylint
-    - python: 3.6
-      env: TOXENV=formatting
     - python: pypy3
       env: TOXENV=pypy
```

Comment:
`pypy3` will be checked by Github actions

Reply:
Nice catch !